ketcher-autotests/tests/File-Management/Open-And-Save-Files/Open-File/Load-Fragment/load-fragment.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { test, expect } from '@playwright/test';
+import {
+  AtomButton,
+  clickInTheMiddleOfTheScreen,
+  pressButton,
+  takeEditorScreenshot,
+  waitForLoad,
+  openFileAndAddToCanvas,
+  openPasteFromClipboard,
+  waitForPageInit,
+  selectAtomInToolbar,
+  selectLeftPanelButton,
+  dragMouseTo,
+  LeftPanelButton,
+  selectDropdownTool,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  moveMouseToTheMiddleOfTheScreen,
+  resetCurrentTool,
+  receiveFileComparisonData,
+  saveToFile,
+} from '@utils';
+import { getKet } from '@utils/formats';
+
+test.describe('load as fragment (Add to Canvas) srtuctures from files with different formats', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - mol-files', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-1854
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await openFileAndAddToCanvas('Molfiles-V2000/cyclopentyl.mol', page);
```

Comment:
Is it supposed to be here that one structure is placed on top of the second? Or is it worth specifying coordinates for the second fragment at a distance from the first structure?

Reply:
It's not placed one on top of another - first structure is automatically moved before adding the second one, that's why I didn't add step for moving elements. 
You can see that it's not overlapped for example here: https://github.com/epam/ketcher/pull/3395/files#diff-2e30bced89687b8f8e40fe4f0276dba03f5ec15ff01fa3a53da1e812a5c43c97